calculators/referendum_supporters_calculator.rb

```diff
@@ -109,8 +110,10 @@ def fetch
 
   private
 
-  def ballot_measure_from_num(bal_num)
-    @ballot_measures.detect { |measure| measure['Measure_number'] == bal_num }
+  def ballot_measure_from_num(election_name, bal_num)
+    @ballot_measures.detect {
+      |measure| measure['election_name'] == election_name &&
```

Comment:
style nitpick, the `|measure|` always goes on the line of the `{` and multi-line blocks use `do`/`end` rather than curly braces. See: https://github.com/rubocop-hq/ruby-style-guide#single-line-blocks